### PR TITLE
update rhai

### PR DESCRIPTION
**Pull request by tabokie**
Signed-off-by: tabokie <xy.tao@outlook.com>

### CODE REVIEW

**Comment by LykxSassinator** [comment]
LGTM

**Comment by LykxSassinator** [comment]
Ping @tabokie Please update this pr.

**Comment by LykxSassinator** [comment]
@glorv PTAL, thx.

File: Cargo.toml [hunk 1]
```diff
@@ -49,7 +49,7 @@ prometheus = { version = "0.13" }
 prometheus-static-metric = "0.5"
 protobuf = "2"
 rayon = "1.5"
-rhai = { version = "1.7", features = ["sync"], optional = true }
+rhai = { version = "1.11", features = ["sync"], optional = true }
```
Comment (tabokie) [inline Cargo.toml:52]:
```suggestion
rhai = { version = "1.7", features = ["sync"], optional = true }
```

**Comment by tabokie** [comment]
@LykxSassinator I meant updating this in your PR.

**Pull request closed without merging**